src/VisualStudio/IntegrationTest/TestUtilities/InProcess/VisualStudioWorkspace_InProc.cs

```diff
@@ -22,14 +22,18 @@ internal class VisualStudioWorkspace_InProc : InProcComponent
 
         private VisualStudioWorkspace_InProc()
         {
+            // we need to enable waiting service before we create workspace
```

Comment:
I think you'd want to move this to somewhere earlier when we are actually setting up the cross-process communication. This might be too late at this point.

Reply:
I confirmed this is early enough. but if you can tell me where is the first point where we start the inproc process, I can put it there as well.